data-protocols/ids/ids-transform-v1/build.gradle.kts

```diff
@@ -23,6 +23,7 @@ plugins {
 dependencies {
     api(project(":spi"))
     api(project(":data-protocols:ids:ids-spi"))
+    implementation(project(":common:util"))
```

Comment:
Just as a note: Seems this dependency has been introduced for making use of `CollectionUtil#isNotEmpty` in `IdsLogicalConstraintToConstraintTransformer`: IMO this could have also been placed directly there since it's just  `collection == null || collection.isEmpty()`.

Reply:
Yes, but it's an internal utility library so I think it is ok and avoids code duplication (albeit one line ;-) )